<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Map</class>
 <widget class="QWidget" name="Map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1800</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGraphicsView {
	background-image: url(:/bg/resourses/images/map.jpg);
}

QPushButton {
	background: rgba(208, 178, 203, 0.7);
	border: 10px white;
	color: white;
}

QPushButton:hover {
    background: rgba(208, 178, 203, 0.5);
}


QPushButton:pressed {
    background: rgb(183, 130, 174);
}

QTimeEdit {
	font: 30px;
	color: rgb(39, 38, 38);
	background: rgba(210, 53, 67, 0.5);
	border: 1px red;
	border-radius: 15px;
}
</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsBtn">
   <property name="geometry">
    <rect>
     <x>1870</x>
     <y>10</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(208, 178, 203);
	font: 35px;
	border: 10px white;
	border-radius: 20px;
	color: white;
}

QPushButton:hover {
    background: rgba(208, 178, 203, 0.5);
}


QPushButton:pressed {
    background: rgb(183, 130, 174);
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="document-properties"/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeBeforeGhost">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="displayFormat">
    <string>ss</string>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>20</second>
    </time>
   </property>
  </widget>
  <widget class="QTimeEdit" name="gameTime">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="displayFormat">
    <string>mm:ss</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="doorHp">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>380</y>
     <width>51</width>
     <height>14</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>154</red>
         <green>153</green>
         <blue>150</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>36</red>
         <green>109</green>
         <blue>32</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>154</red>
         <green>153</green>
         <blue>150</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>36</red>
         <green>109</green>
         <blue>32</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="maximum">
    <number>512</number>
   </property>
   <property name="value">
    <number>512</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="ghostHp">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>51</width>
     <height>14</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>61</red>
         <green>56</green>
         <blue>70</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>165</red>
         <green>29</green>
         <blue>45</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>61</red>
         <green>56</green>
         <blue>70</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>165</red>
         <green>29</green>
         <blue>45</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="maximum">
    <number>512</number>
   </property>
   <property name="value">
    <number>512</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="moneyCount">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="icon">
    <iconset theme="emblem-favorite"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="energyCount">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="icon">
    <iconset theme="multimedia-volume-control"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сдаться</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
